interactive-calculator/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import MyChart from "./Components/SavingsChart";
+import "./App.css";
+
+function App() {
+  function generateData() {}
```

Comment:
Maybe it is just me, but I am a little confused about what this function is doing exactly.

Reply:
Nope, definitely not you :)

I was running behind, moving things around, and must have missed deleting that 

Whoops.

Reply:
That function doesn't do anything.  Author already commented though.